web/styles/zulip.css

```diff
@@ -295,11 +299,14 @@ p.n-margin {
     height: var(--header-height);
     padding-bottom: var(--header-padding-bottom);
     /* Since the headers are sticky, we need non-transparent background. */
-    background-color: hsl(0deg 0% 100%);
+    background-color: var(--color-background);
 }
 
-#top_navbar {
-    border-bottom: 1px solid hsl(0deg 0% 0% / 20%);
+#navbar-middle .column-middle-inner,
+#userlist-toggle-button,
+.header-main,
+#message_view_header {
+    background-color: hsl(0deg 0% 96%);
 }
```

Comment:
This set of items seems a bit random. It might be helpful to both define a variable for this color (maybe `--navbar-background`?) at least; doing so might also save some work in the dark theme CSS file.

Reply:
Ah yeah I worked on this before we started making more constants for colors -- I can add them here.